selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/CheckIntelligenceCommandFromToolbarTest.java

```diff
@@ -85,7 +85,7 @@ public void checkButtonsOnToolbar() {
     checkTestAppAndReturnToIde(currentWindow, "Enter your name:");
     Assert.assertTrue(commandsToolbar.getTimerValue().matches("\\d\\d:\\d\\d"));
     Assert.assertTrue(commandsToolbar.getNumOfProcessCounter().equals("#2"));
-    commandsToolbar.clickOnPreviewCommandBtnAndSelectUrl("ws-machine:8080");
+    commandsToolbar.clickOnPreviewCommandBtnAndSelectUrl("dev-machine:8080");
```

Comment:
@garagatyi: question - should hardcoded port 8080 be maintainable here for the case when Che is started on port which differ from 8080?

Reply:
Sorry I didn't get your question.

Reply:
It is not related to Che master port. Just port of an application which is inside of dev machine

Reply:
I see. Thank you for explanation.